src/content-handlers/iiif/modules/uv-shared-module/css/mixins.less

```
@@ -301,17 +302,7 @@
   }
 
   .directional(@start-color: #555; @end-color: #333; @deg: 45deg) {
-    background-repeat: repeat-x;
-    background-image: -webkit-linear-gradient(
-      @deg,
-      @start-color,
-      @end-color
-    ); // Safari 5.1+, Chrome 10+
-    background-image: -moz-linear-gradient(
-      @deg,
-      @start-color,
-      @end-color
-    ); // FF 3.6+
+    background-repeat: repeat-x; // Safari 5.1+, Chrome 10+ // FF 3.6+
```

Maybe we should lose this comment entirely -- does any of this ancient browser support matter at this stage?

```suggestion
    background-repeat: repeat-x;
```

---

Will remove. A lot of the mixins aren't used from what I can see. I assume it's a common LESS utility file that was added just in case they were needed. When we do a CSS overhaul it'll be nice to clean out all this old stuff :)